sidebars.js

```diff
@@ -1,115 +1,119 @@
-        'self-hosted/deployment/storage/gcs-bucket',
-      ]
-    }
+        "self-hosted/deployment/storage/supported-providers",
+        "self-hosted/deployment/storage/s3-bucket",
+        "self-hosted/deployment/storage/gcs-bucket",
+      ],
+    },
   ],
   product: [
-    { 'Getting Started': ['product'] },
+    { "Getting Started": ["product"] },
     {
-      'Website Live Chat': [
-        'product/channels/live-chat/create-website-channel',
-        'product/channels/live-chat/sdk/setup',
-        'product/channels/live-chat/sdk/identity-validation',
-        'product/channels/live-chat/webflow-integration',
-      ]
+      "Website Live Chat": [
+        "product/channels/live-chat/create-website-channel",
+        "product/channels/live-chat/sdk/setup",
+        "product/channels/live-chat/sdk/identity-validation",
+        "product/channels/live-chat/webflow-integration",
+      ],
     },
     {
-      'API Channel': [
-        'product/channels/api/create-channel',
-        'product/channels/api/send-messages',
-        'product/channels/api/receive-messages'
-      ]
+      "API Channel": [
+        "product/channels/api/create-channel",
+        "product/channels/api/send-messages",
+        "product/channels/api/receive-messages",
+      ],
     },
     {
-      'Others': [
-        'product/channels/facebook',
-        'product/channels/twitter',
-        'product/channels/twilio-whatsapp-sms',
-        'product/channels/supported-features',
-        'product/features/webhooks'
-      ]
-    }
+      Others: [
+        "product/channels/facebook",
+        "product/channels/twitter",
+        "product/channels/twilio-whatsapp-sms",
+        "product/channels/supported-features",
+        "product/features/webhooks",
+      ],
+    },
   ],
   ContributingGuide: [
-    { 'Getting Started': ['contributing-guide'] },
+    { "Getting Started": ["contributing-guide"] },
     {
-      'Environment Setup': [
-        'contributing-guide/environment-setup/mac-os',
-        'contributing-guide/environment-setup/ubuntu',
-        'contributing-guide/environment-setup/windows',
-        'contributing-guide/environment-setup/docker',
-      ]
+      "Environment Setup": [
+        "contributing-guide/environment-setup/mac-os",
+        "contributing-guide/environment-setup/ubuntu",
+        "contributing-guide/environment-setup/windows",
+        "contributing-guide/environment-setup/docker",
+      ],
     },
     {
       "Project Setup": [
-        'contributing-guide/project-setup',
-        'contributing-guide/environment-variables',
-        'contributing-guide/common-errors',
-      ]
+        "contributing-guide/project-setup",
+        "contributing-guide/environment-variables",
+        "contributing-guide/common-errors",
+      ],
+    },
+    {
+      "Mobile App": [
+        "contributing-guide/mobile-app/setup-guide",
+        "contributing-guide/mobile-app/custom_release_guide",
```

Comment:
@muhsin-k Can you please make it `custom-release-guide`? We don't use underscore anywhere in the marketing site?

Reply:
@pranavrajs Done